frontend/src/components/UserGenPage.js

```diff
@@ -51,8 +51,8 @@ class UserGenPage extends Component {
       });
     } else if (window.NativeRobosats && systemClient.getCookie('robot_token')) {
       const token = systemClient.getCookie('robot_token');
-      this.props.setAppState({ token });
-      this.setState({ token, loadingRobot: false });
+      this.setState({ token });
+      this.getGeneratedUser(token);
```

Comment:
this will generate a new `sessionid`, turns out also I had my `sessionid` on Android Studio outdated and this fix worked